python/mxnet/libinfo.py

```diff
@@ -37,6 +38,8 @@ def find_lib_path():
                 logging.warning("MXNET_LIBRARY_PATH should be an absolute path, instead of: %s",
                                 lib_from_env)
             else:
+                if os.name == 'nt':
+                    os.environ['PATH'] = os.path.dirname(lib_from_env) + ';' + os.environ['PATH']
```

Comment:
Please append to the end of path instead of the beginning to prevent path masking